src/introduction/abtnode-setup/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: 'ABT Node Setup'
+description: 'ABT Node Setup'
+keywords: 'abtnode, blocklet'
+author: 'tigeryu8900'
+category: ''
+layout: 'documentation'
+tags:
+  - 'abtnode'
+---
+ABT Node can be installed in the following ways :
+
+  - [Use the binary distribution](#use-the-binary-distribution)
+  - [Use the official Docker image](#use-the-official-docker-image)
```

Comment:
There are more options avaliable to get an ABT Node instance up and running.

1. On AWS: Checkout here: https://www.arcblock.io/en/get-started and https://www.arcblock.io/blog/en/post/2020/09/01/official-guide-to-deploy-abtnode-on-aws

2. Launcher: coming soon, still in beta.

@mave99a Do you think we should add those in this version.

Reply:
Added a section for this